src/main/python/systemds/operator/operation_node.py

```diff
@@ -174,11 +172,15 @@ def write(self, destination: str, format: str = "binary", **kwargs: Dict[str, VA
         unnamed_inputs = [self, f'"{destination}"']
         named_parameters = {"format": f'"{format}"'}
         named_parameters.update(kwargs)
-        return OperationNode(self.sds_context, 'write', unnamed_inputs, named_parameters, output_type=OutputType.NONE)
+        return OperationNode(self.sds_context, 'write', unnamed_inputs, named_parameters)
 
     def print(self, **kwargs: Dict[str, VALID_INPUT_TYPES]) -> 'OperationNode':
         """ Prints the given Operation Node.
         There is no return on calling.
         To get the returned string look at the stdout of SystemDSContext.
         """
-        return OperationNode(self.sds_context, 'print', [self], kwargs, output_type=OutputType.NONE)
+        return OperationNode(self.sds_context, 'print', [self], kwargs)
+
+    def data_type(self):
+        return "operation_node"
+
```

Comment:
can be removed, artefact from an older approach

Reply:
if we can remove it, lets.